signac/contrib/project.py

```diff
@@ -156,9 +169,18 @@ def __setitem__(self, key, value):
             )
 
             assert version.parse(__version__) < version.parse("2.0")
+        if self._mutate_hook is not None:
+            self._mutate_hook()
```

Comment:
Just checking my understanding: this line `self._mutate_hook()` calls the callable that was passed in.

Reply:
Correct. `self._mutate_hook` is a callable, and the `()` calls it with no arguments.